backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -55,6 +55,14 @@ type SendToRail =
   | Rail
   | NoRail
 
+type BinaryOperation =
+  | BinOpAnd
+  | BinOpOr
+
+type Infix =
```

Comment:
Instead of just `Infix`, would `InfixOp` be appropriate? (short for Operation, not Operator)
I've found myself occasionally asking "Infix _what_?"

Reply:
I guess it's sorta "Infix thing"? I'm not sure - I'm not great at naming. But `EInfix` makes sense to me - "Infix Expression".

Reply:
Maybe "the meat of Infix expr"?